src/components/EventListCard/Modal/Preview/EventListCardPreviewModal.tsx

```
@@ -119,9 +119,9 @@ const PreviewModal: React.FC<InterfacePreviewEventModalProps> = ({
             data-testid="updateDescription"
             required
             value={
-              formState.eventdescrip.length > 256
+              formState.eventdescrip?.length > 256
                 ? formState.eventdescrip.substring(0, 256) + '...'
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)